apps/test/unit/lib/levelbuilder/lesson-editor/activitiesEditorReduxTest.js

```diff
@@ -46,6 +49,34 @@ describe('activitiesEditorRedux reducer tests', () => {
     ]);
   });
 
+  it('update tip', () => {
+    const nextState = reducer(
+      initialState,
+      updateTip(1, 2, {
+        key: 'tip-1',
+        type: 'contentCorner',
+        markdown: 'Programming is about solving puzzles.'
+      })
+    ).activities;
+    assert.deepEqual(nextState[0].activitySections[1].tips.map(s => s.type), [
+      'contentCorner'
+    ]);
+    assert.deepEqual(nextState[0].activitySections[1].tips[0], {
+      key: 'tip-1',
+      type: 'contentCorner',
+      markdown: 'Programming is about solving puzzles.'
+    });
+  });
+
+  it('remove tip', () => {
```

Comment:
these tests are good, but please also test at least one case where we do not end up with an empty array, to prove that we are removing the right thing.

Reply:
Updated!